pkg/envinfo/envinfo.go

```diff
@@ -64,7 +64,8 @@ type EnvSpecificInfo struct {
 var _ localConfigProvider.LocalConfigProvider = (*EnvSpecificInfo)(nil)
 
 func (esi EnvSpecificInfo) GetDevfilePath() string {
-	return esi.devfilePath
+	// Return esi.devfilePath without /private as the base folder
+	return strings.Replace(esi.devfilePath, "/private", "", 1)
```

Comment:
This looks really really hacky. We should try to investigate what is happening here and why is this there.